src/providers/okta-helpers.js

```diff
@@ -0,0 +1,22 @@
+const Helpers = {
+  waitAndEmitSAMLResponse: function () {
+    /* eslint-disable  */
+
+    // Stop waiting if an error was found
+    var errEl = document.querySelector('.o-form-has-errors');
+    if (errEl) return true;
+
+    // Keep waiting if no error or SAMLResponse detected
+    var samlEl = document.querySelector('input[name="SAMLResponse"]');
+    if (!samlEl) return false;
```

Comment:
I suppose the calling `wait()` function will handle the retry until the default 30 seconds timeout?